Runtime/BacktraceClient.cs

```diff
@@ -991,7 +991,19 @@ internal void HandleUnhandledExceptionsFromAndroidBackgroundThread(string backgr
             }
             var message = backgroundExceptionMessage.Substring(0, splitIndex);
             var stackTrace = backgroundExceptionMessage.Substring(splitIndex);
-            HandleUnityMessage(message, stackTrace, LogType.Exception);
+            if (Database != null)
+            {
+                Database.Add(new BacktraceReport(new BacktraceUnhandledException(message, stackTrace)).ToBacktraceData(null, GameObjectDepth));
+            }
+            else
+            {
+                HandleUnityMessage(message, stackTrace, LogType.Exception);
```

Comment:
Should we not always `HandleUnityMessage` for background exceptions anymore? Will this behavior be surprising for users?

Reply:
@rqbacktrace ; Konrad told me about this; there is a large chance it cannot be handled because Unity is dying. So prefer handling on startup via the Database.

Reply:
if we have database I think it's safer to use it and send it on the next session or instead prevent application from quitting. Database storage is super fast but things might go wrong with networking AND we won't anyway apply retry settings.